src/rules/prettier/apply.test.ts

```diff
@@ -0,0 +1,26 @@
+import { Prettier } from '.';
+
+const rootPath = './prettier';
+
+const util = require('util');
+jest.mock('util');
+
+const cp = require('child_process');
+jest.mock('child_process');
+
+test('Should run npm i prettier -DE command', () => {
+  jest.mock(rootPath + 'package.json', () => ({}), { virtual: true });
+
+  const prettier = new Prettier(rootPath);
+
+  util.promisify.mockImplementation((exec: (cmd: string) => {}) => {
```

Comment:
Why you did not use the same structure as the test for the linter rule I have updated few days ago ?